docs/uninstallation.md

```diff
@@ -0,0 +1,37 @@
+# Uninstalling NGINX Kubernetes Gateway
```

Comment:
I wonder if we need a separate doc for that.

What if we include it as a section in the installation doc?

This way we don't need to have  Prerequisites section (installation doc already has it). And we don't need to have the git clone part (installation doc already has it)

In my opinion, deinstallation is tightly related to installation, and since it is very simple for NKG, I think it makes sense to have it in the same doc as the installation instructions.